notify/impl.go

```diff
@@ -30,7 +30,7 @@ import (
 	"net/url"
 	"strings"
 	"time"
-
+    "io/ioutil"
```

Comment:
there needs to be a linebreak between the stdlib packages and the third-party github packages. So, just add a newline below this.